### PR TITLE
Add manifest for org.kde.Ikona

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 
   - https://invent.kde.org/sdk/ikona/-/merge_requests/8

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.Ikona

File: org.kde.Ikona.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "build-options": {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```
Comment (hfiguiere) [inline org.kde.Ikona.json:20]: this won't work. There is not network access.

File: org.kde.Ikona.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
```
Comment (hfiguiere) [inline org.kde.Ikona.json:22]: "Ikona" is the module that uses rust, this won't work.

File: org.kde.Ikona.json (hunk 1 above)
Comment (hfiguiere) [reply org.kde.Ikona.json:20]:
```
Error: Can't specify build-args in sandboxed build
```

Comment (flyingcakes85) [reply org.kde.Ikona.json:20]: Rust dependencies are downloaded at compile time, so how can I achieve that here? I'll have to look at manifests of other Rust applications on Flathub.

File: org.kde.Ikona.json (hunk 2 above)
Comment (hfiguiere) [reply org.kde.Ikona.json:22]: like python or node

https://github.com/flatpak/flatpak-builder-tools

File: org.kde.Ikona.json (hunk 1 above)
Comment (hfiguiere) [reply org.kde.Ikona.json:20]: like python or node

https://github.com/flatpak/flatpak-builder-tools

**Comment by flyingcakes85** [comment]
bot, build org.kde.Ikona

**Comment by flyingcakes85** [comment]
@hfiguiere I have updated the manifest

File: org.kde.Ikona.json (hunk 1 above)
Comment (flyingcakes85) [reply org.kde.Ikona.json]: Thanks for the pointer :)

**Pull request closed without merging**